modules/mixed-arch-import-imagestreams.adoc

```diff
@@ -0,0 +1,74 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-import-imagestreams_{context}"]
+
+= Importing manifest lists in image streams on your heterogeneous cluster 
+
+On a {product-title} {product-version} heterogeneous cluster, the image streams in the cluster do not import manifest lists automatically. You must manually change the default `importMode` option in order to import the manifest list.
+
```

Comment:
should we mention that the importMode should be changed to "PreserveOriginal" or is that redundant because it is contained in the IR docs? also should we reference the IR doc pertaining to importMode here?

Reply:
The IR docs are about image stream imports so that could be confusing.

Reply:
Maybe I can change it to something like this 
```suggestion
On a {product-title} {product-version} heterogeneous cluster, the image streams in the cluster do not import manifest lists automatically. You must manually change the default `importMode` option to `PerserveOriginal` in order to import the manifest list.
```

Reply:
Looks great @kelbrown20!

Reply:
Just updated!